docs/fundamentals/networking/http/httpclient-migrate-from-httpwebrequest.md

```diff
@@ -0,0 +1,692 @@
+---
+title: Migrate from HttpWebRequest
+description: Guidance document for migrating from HttpWebRequest to HttpClient.
+author: liveans
+ms.author: aaksoy
+ms.date: 07/25/2024
+helpviewer_keywords: 
+  - "protocols, HTTP"
+  - "migration, HTTP"
+  - "HTTP"
+  - "HttpWebRequest class, about HttpWebRequest class"
+  - "application protocols, HTTP"
+  - "HttpClient class, about HttpClient class"
+  - "data requests, HTTP"
+  - "Internet, HTTP"
+---
+
+# HttpWebRequest to HttpClient Migration Guide
+
+This document aims to guide developers through the process of migrating from <xref:System.Net.HttpWebRequest>, <xref:System.Net.ServicePoint>, and <xref:System.Net.ServicePointManager> to <xref:System.Net.Http.HttpClient>. The migration is necessary due to the obsolescence of the older APIs and the numerous benefits offered by <xref:System.Net.Http.HttpClient>, including improved performance, better resource management, and a more modern and flexible API design. By following the steps outlined in this document, developers will be able to transition their codebases smoothly and take full advantage of the features provided by <xref:System.Net.Http.HttpClient>.
```

Comment:
While mentioning performance, should we mention that it's not just a "nice to have" bump in perf by migrating.
It's really that their existing WebRequest logic's perf is likely going to completely tank once they move to Core since WebRequest is just a minimal compat layer (e.g. no connection reuse in tons of cases).

Reply:
I was actually thinking to add a note or warning for this, to make sure that people who reads this doc will be aware of that `WebRequest` is compat layer on top of `HttpClient`.